app/javascript/components/reusable/withUserForm.js

```diff
@@ -316,7 +316,7 @@ const withUserForm = (WrappedComponent, schema, wrappedProps) => {
                       />
                     )
                   }
-                  value={ country ? country : 'United States' }
```

Comment:
Great fix! I am curious about what makes this work though, is it because JS is reading the empty string as a truthy value and that's why this ternary originally didn't work? Otherwise great!

Reply:
Yes, in JS empty string is falsey value.